canonical-web-and-design/jaas-dashboard
# Align table columns

## Done

- Align table columns
- Switch machine and unit icons in config column
- Capitalise td strings to match designs
- Turn off Vanilla text increases at larger breakpoints
- Tweak breakpoint for topology panel to 1580px

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- View models and model details tables and verify that columns align as per the design

If you add this snippet to `index.scss`, you can see the column alignment visually.

```css
td {border-right: 1px solid orangered}
```

Fixes #119, 
Fixes #140, 
Fixes #142, 
Fixes #180, 
Fixes #146,
Fixes #179
